assets/src/components/presets.tsx

```diff
@@ -1,15 +1,34 @@
 import React, { useContext } from "react"
 import { createPreset, instantiatePreset, savePreset } from "../state"
+import CloseButton from "./closeButton"
 import { StateDispatchContext } from "../contexts/stateDispatchContext"
 import { currentRouteTab, isPreset, isEditedPreset } from "../models/routeTab"
+import { plusThinIcon } from "../helpers/icon"
 
 const Presets = () => {
   const [{ routeTabs }, dispatch] = useContext(StateDispatchContext)
   const presets = routeTabs.filter(isPreset)
 
   return (
     <div className="m-presets-panel">
+      <ul>
+        {presets.map((preset) => (
+          <li key={preset.uuid}>
+            <div className="m-presets-panel__preset-button-container">
```

Comment:
**question:** could you assign this class to the button instead, or is there a reason you need a wrapping div?

Reply:
I wanted `flex-grow: 1` to apply to the container but not to the button, because if it applies to the button than the entire empty horizontal space between the preset title and the "x" button is a clickable part of the preset name.

Reply:
Wouldn't it be good to have as big a click target as possible? I'm remembering this discussion about a similar thing from last month: https://mbta.slack.com/archives/C10EPAU6Q/p1642090431014400?thread_ts=1642088728.010000&cid=C10EPAU6Q

If not, `justify-content: space-between` might do the spacing without requiring `flex-grow` or a wrapper.

Reply:
Ah, good point about `space-between`, that probably would work.

Regarding the click area itself, Tyler is going to take a look at this and I'll see what he has to say.

Reply:
Okay well I tried this and discovered the hard way that there is another reason this container was helping me out: the close button uses `currentColor` to match its surrounding text, and having a sibling with the blue color we're using was causing the close button to also be blue. Of course I can override that, but then because of selector specificity I have to either copy-paste the `:focus` and `:hover` states into this CSS or resort to hacks like `!important`. I'm open to other ideas but unless I think of something else I think I'm going to keep the container.

Reply:
I can at least use `justify-content: space-between` on the `li` instead of `flex-grow: 1` on the container though, so there's that. 😅